Gemfile

```diff
@@ -50,3 +52,10 @@ group :development do
   gem 'spring'
 end
 
+group :test do
+  # Adds support for Capybara system testing and selenium driver
+  gem 'capybara', '>= 2.15'
+  gem 'poltergeist'
```

Comment:
Could you explain why you're requiring poltergeist here? It seems to be an older, no longer supported gem that isn't needed for capybara to work.

Reply:
From the class we took with Ramiro poltergeist is used to communicate to make phantonjs headless browser function.

Reply:
https://github.com/teamcapybara/capybara#poltergeist
Is a link to it, It just sets up a headless browser with phantomjs headless browser.